internal/exec/helmfile_generate_varfile.go

```diff
@@ -0,0 +1,82 @@
+package exec
+
+import (
+	"errors"
+	"fmt"
+	c "github.com/cloudposse/atmos/pkg/config"
+	"github.com/cloudposse/atmos/pkg/utils"
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"path"
+)
+
+// ExecuteHelmfileGenerateVarfile executes `helmfile generate varfile` command
+func ExecuteHelmfileGenerateVarfile(cmd *cobra.Command, args []string) error {
+	if len(args) != 1 {
+		return errors.New("invalid arguments. The command requires one argument `component`")
+	}
+
+	flags := cmd.Flags()
+
+	stack, err := flags.GetString("stack")
+	if err != nil {
+		return err
+	}
+
+	component := args[0]
+
+	var info c.ConfigAndStacksInfo
+	info.ComponentFromArg = component
+	info.Stack = stack
+	info.ComponentType = "helmfile"
+
+	info, err = ProcessStacks(info)
```

Comment:
If this outputs all the yaml for all the deep merged stacks, I'd love to see the output of ProcessStacks with its own subcommand so I can quickly grep for a specific component.

Reply:
we can add a separate command for that